src/client/pages/matchPage/MatchPage.jsx

```diff
@@ -1,53 +1,118 @@
 import React from 'react';
-//import { useState } from 'react';
+import { useState, useEffect } from 'react';
+import { GET_BOOKS_URL } from '../../config';
+import { generateRandomInt } from '../../utils';
 
 import Footer from '../../components/Footer'
 
 import '../../styles/match.css'
 
 const MatchPage = () => {
-    // const [book, setBook] = useState({});
+    const [library, setLibrary] = useState({});
+    const [book, setBook] = useState({});
+    const [displayMatchCard, setDisplayMatchCard] = useState(true);
+    const [recommendations, setRecommendations] = useState([]);
+
+    const pickRandomBook = (library) => {
+        const generatedIndex = generateRandomInt(library.length - 1);
```

Comment:
What happens here if `library.length` is 0? You'd effectively be doing `generateRandomInt(-1)` - how would that affect your app?

Reply:
You have a strong point here. Will modify